dev/com.ibm.ws.security.common/src/com/ibm/ws/security/common/token/propagation/TokenPropagationHelper.java

```diff
@@ -275,7 +275,7 @@ public static synchronized boolean pushSubject(String username) {
             FFDCFilter.processException(e,
                     TokenPropagationHelper.class.getName(), "pushSubject",
                     new Object[] { username });
-            Tr.error(tc, "ERROR_AUTHENTICATE", new Object[] { e.getMessage() }); // CWWKS6103E
+            Tr.error(tc, "ERROR_AUTHENTICATE", new Object[] { e }); // CWWKS6103E
```

Comment:
The insert for this message actually looks like it should be a username, not an exception. The message is
```
ERROR_AUTHENTICATE=CWWKS6103E: A security subject cannot be created for the {0} user.
```
I'm trying to decide if we should enhance the message to add an insert at the end that could be the Exception object.